site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -0,0 +1,199 @@
+`<area>` and `<input>` elements.
+
+```webidl
+interface mixin InvokerElement {
+  [CEReactions] attribute Element? invokeTargetElement;
+  [CEReactions] attribute DOMString invokeAction;
+};
+HTMLButtonElement extends InvokerElement
+HTMLInputElement extends InvokerElement
+```
+
+The `interesttarget` value should be an IDREF pointing to an element within the
+document. `.interestTargetElement` also exists on the element to imperatively
+assign a node to be the invoker target, allowing for cross-root invokers (in
+some cases, see [the popovertarget attr-asociated element
+steps](https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#attr-associated-element)
+for more).
+
+Elements with `interesttarget` set will dispatch an `InterestEvent` on the
+_Interestee_ (the element referenced by `interesttarget`) when the element
+_Shows Interest_ or _Loses Interest_. When the element _Shows Interest_ the
+event type will be `'interest'`. If the element has _Shown Interest_, and
+interest moves away from both the _Interest Element_ and the _Interestee_, then
+the element _Loses Interest_ and an `InterestEvent` with the type of
+`'loseinterest'` will be dispatched on the _Interestee_. The event also contains
+a `relatedTarget` property that will reference the _Interest_ element.
+`InterestEvents` are always non-bubbling, cancellable events.
+
+```webidl
+[Exposed=Window]
+interface InterestEvent : Event {
+  constructor(DOMString type, InterestEventInit interestEventInit);
+  readonly attribute Element invoker;
+};
+dictionary InterestEventInit : EventInit {
+  Element invoker;
+};
+```
+
+Both `interesttarget` and `invoketarget` can exist on the same element at the
+same time, and both should be respected.
+
+If a `<button>` is a form participant, or has `type=submit`, then `invoketarget`
+_must_ be ignored. `interesttarget` is still valid in these scenarios.
+
+If an `<input>` is a form participant, or has a `type` other than `reset` or
+`button`, then `invoketarget` and `interesttarget` _must_ be ignored. `interesttarget`
+additionally works with `type` of `submit`.
+
+### Example Code
+
+#### Popovers
+
+An `interesttarget` allows for tooltips using `popover`:
+
+```html
+<button interesttarget="my-popover">Open Popover</button>
+
+<div id="my-popover" popover="hint">Hello world</div>
+```
+
+#### Custom behaviour
+
+Elements with _Interest_ will dispatch events on the _Interestee_ element,
+allowing for custom JavaScript to be triggered without having to wire up manual
+event handlers to the Interest elements.
+
+```html
+<button interesttarget="my-custom">
+  When this button shows interest, the below div will display.
+</button>
+
+<div id="my-custom" hidden>Supplementary information</div>
+
+<script>
+  const custom = document.getElementById("my-custom");
+  custom.addEventListener("interest", (e) => {
+    custom.hidden = false;
+  });
+  custom.addEventListener("loseinterest", (e) => {
+    custom.hidden = true;
+  });
+</script>
+```
```

Comment:
I'm guessing the a11y of this example is "bad". It's ok to include, but I bet you'll get comments. I wonder if there's something else you can do other than making content visible and invisible? Playing a sound or something?